lib/web_ui/lib/src/engine/semantics/image.dart

```diff
@@ -55,6 +55,7 @@ class ImageRoleManager extends RoleManager {
 
   void _setLabel(html.Element element) {
     if (semanticsObject.hasLabel) {
+      print('>>> semanticsObject.label = ${semanticsObject.label}');
```

Comment:
Dead code?

Reply:
And a few more below.

Reply:
This is just a draft PR. I exported it early to help reproduce https://github.com/dart-lang/sdk/issues/42324. Sorry if it wasn't clear.

Reply:
Cleaned it up.